test/modules/packages/mpi/multilocale/hello-chapel-nl-1/SKIPIF

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+
+
+import os
+retval = True
+
+# Only support the GNU compilers for now
+compiler=os.environ['CHPL_TARGET_COMPILER']
+retval = (compiler !="mpi-gnu") & (compiler!="cray-prgenv-gnu")
+
+# Only support fifo 
+retval |= (os.environ['CHPL_TASKS']!="fifo")
+
+# Only support GASNET+MPI 
+retval |= (os.environ['CHPL_COMM']!="gasnet")
+
+if ('CHPL_COMM_SUBSTRATE' not in os.environ) :
```

Comment:
After #4216 is merged, this env var should always be defined during a `start_test` run.